### PR TITLE
docs: IDE Setup IntelliJ + K8s Plugin

### PR DESCRIPTION
**Pull request by sernst**
For users with the JetBrains Kubernetes plugin installed, there's
an alternate IDE configuration path that can be both beneficial
and avoid a conflicting validation state when both methods are
enabled on the same file.

### CODE REVIEW

File: docs/ide-setup.md [hunk 1]
```diff
@@ -20,6 +20,20 @@ That's it. Open an Argo YAML file and you should see smarter behaviour, includin
 
 ![IDEA Example Functionality](assets/intellij-ide-step-1-example-functionality.png)
 
+### IntelliJ IDEA (Community & Utimate Editions) + Kubernetes Plugin
```
Comment (terrytangyuan) [inline docs/ide-setup.md:23]: This should work for GoLand as well.

Comment (sernst) [reply docs/ide-setup.md:23]: Yep, agreed. And every other JetBrains IDE as well. 

I'm certainly happy to change the language on that to be broader in terms of JetBrains IDE support. However, I'd advocate for changing the previous header to be consistent as well in that case:

https://github.com/argoproj/argo-workflows/blob/master/docs/ide-setup.md?plain=1#L7

Thoughts?

Comment (terrytangyuan) [reply docs/ide-setup.md:23]: Sounds good to me

Comment (sernst) [reply docs/ide-setup.md]: Great. I've made the change.

**Review by terrytangyuan — APPROVED** [review]
Thank you!

**Pull request merged**